src/v2/Apps/Settings/Routes/EditSettings/Components/SettingsEditSettingsTwoFactor/TwoFactorAuthentication/Components/AppSecondFactor/index.tsx

```diff
@@ -59,8 +68,8 @@ export const AppSecondFactor: React.FC<AppSecondFactorProps> = props => {
       setShowCompleteRedirectModal(true)
     }
 
-    if (props.me.hasSecondFactorEnabled) {
-      relayRefetch?.refetch({}, {}, showCompleteModalCallback)
+    if (me.hasSecondFactorEnabled) {
+      relay.refetch({}, {}, showCompleteModalCallback)
```

Comment:
One minor thing I changed: previously these would accept a prop from the parent component to refetch _that_ component. I switched them to just be local refetches, which makes more sense. What would actually make sense though is not refetching at all and returning a `me` in the fields so you can simply return the correct data along with the mutation. It seems like this is not a commonly known thing that you can do?

Reply:
I think it's pretty well known? But i could be wrong.